libs/events/src/types/events.ts

```diff
@@ -0,0 +1,13 @@
+import { OnExecutedOrderPayload, OnPostedOrderPayload, OnRejectedOrderPayload } from './orders'
+import { OnToastMessagePayload } from './toastMessages'
+
+export type EventNames = 'updateWidgetConfig' | 'onToastMessage' | 'onPostedOrder' | 'onExecutedOrder'
+
+// Define types for event payloads
+export interface EventPayloads {
+  updateWidgetConfig: { theme: string } // TODO: move here the widget params?
```

Comment:
don't pay too much attention on my exact pick for what events we have. 

The main point here, we can define the exact messages we allow, and we link each message with its payload type.